client/js/socket-events/quit.js

```diff
@@ -1,12 +1,20 @@
 "use strict";
 
 const $ = require("jquery");
+const chat = $("#chat");
 const socket = require("../socket");
 const sidebar = $("#sidebar");
 
 socket.on("quit", function(data) {
 	const id = data.network;
-	sidebar.find("#network-" + id).remove();
+	const network = sidebar.find("#network-" + id);
+
+	network.children().each(function() {
+		// this = child
+		chat.find(this.getAttribute("data-target")).remove();
```

Comment:
Use `$(this).data("target")`

Reply:
Was thinking it was unnecessary to use jQuery for this, but sure :+1: